CharacterFactory.py

```diff
@@ -0,0 +1,108 @@
+import copy
+
+
+class CharacterFactory:
+    def __init__(self):
+        pass
+
+    def createCharacter(
+        self, name, classType
+    ):  # creates a character with the given name and classType
+        char = Character(name, classType)
+        return char
+
+
+# Dictionary of class types
+# [name, classType, health, magic, mechanical, defense, magicDefense, level, loot, food ]
+# basic class types, can be tweaked and added to as needed
+class_type = {
+    "blank": ["blank", 100, 0, 0, 0, 0, 0, 1, [], []],
+    "warrior": ["warrior", 100, 0, 15, 0, 0, 1, [], []],
+    "mage": ["mage", 100, 15, 0, 0, 0, 1, [], []],
+    "thief": ["thief", 100, 0, 10, 0, 0, 1, [], []],
+}
```

Comment:
Rename this to `class_types` and move it into the factory.

Reply:
It could also be a good idea to have a method like `get_class_types` that returns a list of all the classes. Not sure where this would be used but it sounds like it could be useful.